scripts/add_qm_metrics.py

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""
+Script to add quality metrics file to M&C database.
+"""
+
+import hera_mc.mc as mc
+import os
+import numpy as np
+import warnings
+
+
+parser = mc.get_mc_argument_parser()
+parser.add_argument('files', metavar='files', type=str, nargs='+',
+                    help='json files to read and enter into db.')
+parser.add_argument('--type', dest='type', type=str, default=None,
+                    help='File type to add to db. Options = ["ant", "firstcal", "omnical"]')
+args = parser.parse_args()
+db = mc.connect_to_mc_db(args)
+session = db.sessionmaker()
+
+files = args.files
+if len(files) == 0:
+    raise AssertionError('Please provide a list of quality metric files.')
```

Comment:
Since this is a standalone script, we know that an exception here is going to end up printing distracting backtrace information. Not a huge deal but you'd get cleaner output with:

```
import sys
print >>sys.stderr, 'Please provide...' # since this module doesn't use the print_function __future__
sys.exit(1)
```